experiments/tester/src/RNTester/TestComponents/Text/TextTest.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { Separator } from '@fluentui-react-native/separator';
+import { StandardUsage } from './StandardUsage';
+import { CustomizeUsage } from './CustomizeUsage';
+import { styles } from './styles';
+import { TextWin32 } from '@office-iss/react-native-win32';
+
+export const TextTest: React.FunctionComponent<{}> = () => {
+  return (
+    <View>
+      <TextWin32 style={styles.section}>Standard Usage</TextWin32>
```

Comment:
Does this need to be `TextWin32`. I think it can just be normal `Text`, right?

Reply:
Yeah that should be fine. I'll change it.